### PR TITLE
[Core] Add the two titanshard trinkets + setbonus from raden

### PR DESCRIPTION
**Pull request by Pewtro**
image

### CODE REVIEW

File: src/parser/core/CombatLogParser.js [hunk 1]
```diff
@@ -175,20 +165,21 @@ import StormglideSteps from '../shared/modules/items/bfa/raids/crucibleofstorms/
 import TridentOfDeepOcean from '../shared/modules/items/bfa/raids/crucibleofstorms/TridentOfDeepOcean';
 import VoidStone from '../shared/modules/items/bfa/raids/crucibleofstorms/VoidStone';
 import ZaxasjsDeepstriders from '../shared/modules/items/bfa/raids/crucibleofstorms/ZaxasjsDeepstriders';
-// Azsharas Eternal Palace
```
Comment (niseko) [inline src/parser/core/CombatLogParser.js]: poor Eternal Palace

File: src/parser/shared/modules/items/bfa/raids/nyalothathewakingcity/VoidTwistedTitanshard.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  };
+
+  critRating = null;
+  procs = 0;
+
+  constructor(...args) {
+    super(...args);
+    this._item = this.selectedCombatant.getTrinket(ITEMS.VOID_TWISTED_TITANSHARD.id);
+    this.active = !!this._item;
+    console.log(this._item);
```
Comment (niseko) [inline src/parser/shared/modules/items/bfa/raids/nyalothathewakingcity/VoidTwistedTitanshard.js]:
```suggestion
```

File: src/parser/shared/modules/items/bfa/raids/nyalothathewakingcity/VoidTwistedTitanshard.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    this.procs += 1;
+  }
+
+  statistic() {
+    return (
+      <Statistic
+        category={STATISTIC_CATEGORY.ITEMS}
+        size="flexible"
+        tooltip={(
+          <> :) </>
```
Comment (niseko) [inline src/parser/shared/modules/items/bfa/raids/nyalothathewakingcity/VoidTwistedTitanshard.js]: is there supposed to be something here?

File: src/parser/shared/modules/items/bfa/raids/nyalothathewakingcity/VoidTwistedTitanshard.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        )}
+      >
+        <div className="pad">
+          <label><ItemLink id={ITEMS.VOID_TWISTED_TITANSHARD.id} details={this._item} /></label>
+
+          <div className="value">
+            <UptimeIcon /> {formatPercentage(this.uptime, 0)}% <small>uptime</small><br />
+            <UptimeIcon /> {formatPercentage(this.effectiveness, 0)}% <small>effectiveness</small><br />
```
Comment (niseko) [inline src/parser/shared/modules/items/bfa/raids/nyalothathewakingcity/VoidTwistedTitanshard.js]: hps of the shield would be interesting

File: src/parser/shared/modules/items/bfa/raids/nyalothathewakingcity/VoidTwistedTitanshard.js (hunk 3 above)
Comment (Pewtro) [reply src/parser/shared/modules/items/bfa/raids/nyalothathewakingcity/VoidTwistedTitanshard.js]: O, I forgot to commit

File: src/parser/core/CombatLogParser.js (hunk 1 above)
Comment (Pewtro) [reply src/parser/core/CombatLogParser.js]: Yeah I am killing off one of them - we have 2x TEP, next commits cleans it up a bit more.

File: src/parser/shared/modules/items/bfa/raids/nyalothathewakingcity/VoidTwistedTitanshard.js (hunk 4 above)
Comment (Pewtro) [reply src/parser/shared/modules/items/bfa/raids/nyalothathewakingcity/VoidTwistedTitanshard.js]: Good idea, I'll move effectiveness and add HPS

File: src/parser/shared/modules/items/bfa/raids/nyalothathewakingcity/VoidTwistedTitanshard.js [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+          <>Effectiveness refers to how long the shield persisted on average out of the maximum duration.</>
+        )}
+      >
+        <div className="pad">
+          <label><ItemLink id={ITEMS.VOID_TWISTED_TITANSHARD.id} details={this._item} /></label>
+
+          <div className="value">
+            <UptimeIcon /> {formatPercentage(this.uptime, 0)}% <small>uptime, {formatPercentage(this.effectiveness, 0)}% effectiveness</small><br />
+            <HasteIcon /> {formatNumber(this.averageCritRating)} <small>average Crit gained</small><br />
+            <HealthIcon />{formatNumber(this.hps)} <small>HPS</small>
```
Comment (niseko) [inline src/parser/shared/modules/items/bfa/raids/nyalothathewakingcity/VoidTwistedTitanshard.js]:
```suggestion
            <ItemHealingDone amount={this.healing} />
```
from `import ItemHealingDone from 'interface/ItemHealingDone';`

Would be more consistent with the other statistics.

**Pull request merged**